Development.podspec

```diff
@@ -15,6 +15,7 @@ Pod::Spec.new do |s|
   s.dependency 'WorkflowRxSwift'
   # s.dependency 'WorkflowCombine' # TODO: Disabled because app specs cannot increase the deployment target of the root
```

Comment:
Unrelated, but I bet we can turn these on globally now cc @jamieQ

Reply:
I don't believe we can if this is to work around a bug in `pod gen` unless we're bumping all podspecs to iOS 14.

The project attempts to generate with an iOS 11 deployment target and fails to resolve all targets as a result (since these commented ones require iOS 14).  I just reported the bug in #cocoapods [here](https://square.slack.com/archives/C09R3231Q/p1675121889596839).

Reply:
I figured we could bump stuff in the whole repo to 14 as well. Anyway, not important for this PR!

Reply:
I'd expect that you'd be able generate a project with pod gen which ends up just using the _highest_ deployment target found in your development podspecs, but it doesn't appear to work this way (it's either an order thing or choosing the lowest version I think).

Reply:
looking into bumping the versions [here](https://github.com/square/workflow-swift/pull/186). can do that after this is merged though & will update the values in the new podspec.